src/main/java/org/fundacionjala/pivotal/pageobjects/dashboard/FormCreate.java

```diff
@@ -116,6 +116,7 @@ public void setProjectAccount(final String accountName) {
      * @param value .
      */
     public void setProjectPrivacy(final String value) {
+        //TODO change to if
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)